set-dotnet-versions.bat

```diff
@@ -8,13 +8,13 @@ set version_postfix=
 
```

Comment:
You should also bump the patch above

Reply:
🤦 oops, thanks, done in https://github.com/DataDog/datadog-aas-extension/pull/71/commits/12774fbaaf8216ffa96a353b0132037c6c07dd2f